builder/actions/cmake.py

```diff
@@ -246,7 +246,7 @@ def run(self, env):
             return
 
         ctest = toolchain.ctest_binary()
-        sh.exec(*toolchain.shell_env, ctest,
+        sh.exec(*toolchain.shell_env, ctest, "--parallel", str(os.cpu_count()),
```

Comment:
Why?  If this is affecting all repos then that's very concerning.

Reply:
the tests are slow. Hmmm, do we have tests that cannot run in parallel? If so, I guess I'll just change s3 to run in parallel instead

Reply:
I worry about crt-cpp where we have mqtt tests that used unique client ids but share topics and cross-talk between them

Reply:
Ideally all tests would be truly independent and mutually exclusive, but I don't think we're there yet.

Reply:
Sure, I'll just enable it only for aws-c-s3 then.

Reply:
Still needs to update builder for the script to import environment variables